app/components/Views/DetectedTokens/index.tsx

```diff
@@ -65,11 +66,7 @@ interface IgnoredTokensByAddress {
 const DetectedTokens = () => {
   const navigation = useNavigation();
   const sheetRef = useRef<SheetBottomRef>(null);
-  const detectedTokens = useSelector<any, TokenType[]>(
-    (state) =>
-      state.engine.backgroundState.TokensController
-        .detectedTokens as TokenType[],
-  );
+  const detectedTokens = useSelector<any, TokenType[]>(selectDetectedTokens);
```

Comment:
These generic parameters don't seem to be necessary:

```suggestion
  const detectedTokens = useSelector(selectDetectedTokens);
```

Seems worth getting rid of this especially given the use of `any` here